README.md

```diff
@@ -46,25 +46,33 @@ Citus Cloud runs on top of AWS as a fully managed database as a service and has
 
 If you're looking to get started locally, you can follow the following steps to get up and running.
 
-* Install docker-compose: [Mac][mac_install] | [Linux][linux_install]
-* (Mac only) connect to Docker VM
-  ```bash
-  eval $(docker-machine env default)
-  ```
-
-* Pull and start the docker images
+1. Install Docker Community Edition and docker-compose
+  * Mac:
+    1. [Download](https://www.docker.com/community-edition#/download) and install Docker.
+    2. Start Docker by clicking on the application’s icon.
+  * Linux:
+    ```bash
+    curl -sSL https://get.docker.com/ | sudo bash
+    sudo usermod -aG docker $USER && newgrp docker
+    sudo systemctl start docker
+
+    sudo curl -L https://github.com/docker/compose/releases/download/1.11.2/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
+    sudo chmod +x /usr/local/bin/docker-compose
+    ```
+    (This docker-compose version is fine for running Citus, or you can install the [latest version](https://github.com/docker/compose/releases/latest).)
+1. Pull and start the docker images
   ```bash
   wget https://raw.githubusercontent.com/citusdata/docker/master/docker-compose.yml
```

Comment:
It's annoying to have `wget` and `curl` in the same script. Use `curl -sSLO` instead.